frontend/services/entities/software.ts

```diff
@@ -234,7 +234,39 @@ export default {
       true
     );
   },
+  // TODO: Only send data that is changing and test E2E
+  editSoftwarePackage: (
+    softwareId: number,
+    data: IAddPackageFormData,
+    timeout?: number
+  ) => {
+    const { SOFTWARE_TITLE } = endpoints;
+
+    if (!data.software) {
+      throw new Error("Software package is required");
+    }
 
+    const formData = new FormData();
+    formData.append("software", data.software);
+    formData.append("self_service", data.selfService.toString());
+    data.installScript && formData.append("install_script", data.installScript);
+    data.preInstallQuery &&
+      formData.append("pre_install_query", data.preInstallQuery);
+    data.postInstallScript &&
+      formData.append("post_install_script", data.postInstallScript);
+    // TODO: Add uninstall script once Jacob's 20320 uninstall work is merged
+    // data.uninstallScript &&
+    //   formData.append("uninstall_script", data.uninstallScript);
```

Comment:
@RachelElysia We're now rebased on top of Jacob's work, so we should be able to uncomment this?

Reply:
Yes please!